calendar/page.js

```diff
@@ -26,12 +26,17 @@ function isRecordValid(record) {
   typeof record.title === 'string'
 }
 
+function getMonthName(){
+  return calendarHandler.calendar.getDate().d.d.toLocaleString('en-us', { month: 'long', year:'numeric' })
```

Comment:
what is .d.d?

Reply:
most grist code leaves a space after a colon

Reply:
and a space before a curly brace starting a function (like `getMonthName() {`)

Reply:
> what is .d.d?

TUI Calendar is using custom TZDate structure to handle the dateTime and  i used [..].d.d.[...] to access plain old Date format. Nerveless i found more canonical way to do this is documentation :)